crypto/transactions/vote.py

```diff
@@ -0,0 +1,22 @@
+from crypto.constants import TRANSACTION_FEES, TRANSACTION_VOTE
+from crypto.transactions.base import BaseTransaction
+
+
+class VoteTransaction(BaseTransaction):
+
+    transaction_type = TRANSACTION_VOTE
+
+    def __init__(self, vote, fee=None):
+        """Summary
+
+        Args:
+            vote (str): address of a delegate you want to vote
+            fee (int, optional): fee used for the transaction (default is already set)
+        """
+        super().__init__()
+        self.asset['votes'] = [vote]
+        self.fee = fee or TRANSACTION_FEES[self.transaction_type]
```

Comment:
Since you repeat this in all subclasses (except one has modified value), would it make sense to put this into base class and only override it where needed?

Reply:
as discussed via DM, this can't be done because `transaction_type` is not available directly inside the base class